source/tutorials/tutorials.rst

```diff
@@ -7,6 +7,18 @@ This section provides tutorials on getting started with NanoVer.
 We assume familiarity with setting up simulations in whichever 
 framework you're running with NanoVer.
 
+The tutorials use `Jupyter notebooks <https://jupyter.org>`_, `NGLView <https://github.com/nglviewer/nglview>`_ for visualising trajectories, and while not strictly necessary, assumes you have the `NanoVer IMD VR <https://gitlab.com/intangiblerealities/nanover-applications/nanover-imd>`_ application installed. These can all be installed with conda:
```

Comment:
The VR client is called `NanoVer iMD` not `NanoVer IMD VR`. If you want to clarify that it is the VR client, which may very much be needed, you'll have to phrase it differently.

Reply:
Ah I see! I think I just copied this across from the README of the nanover-protocol repo, so if this needs changing it should probably be changed in both places. Is the link used above the correct link for `NanoVer iMD`?

Reply:
Good catch, it is an old link. Here is the correct one: https://github.com/irl2/nanover-imd

Reply:
Awesome, thanks! I'll update that now, and add something to clarify that this is the VR client